src/steps/upgrade.vim

```diff
@@ -33,16 +33,16 @@ if exists(":PaqUpdate")
     PaqUpdate
 endif
 
-if exists(":Lazy")
-    echo "Lazy Update"
-    Lazy! sync | qa
-endif
-
 if exists(":AstroUpdate")
    echo "AstroUpdate"
    AstroUpdate
 endif
 
+if exists(":Lazy")
+    echo "Lazy Update"
+    Lazy! sync | qa
+endif
+
```

Comment:
Would you like to add some comments stating the rationale for this change in case someone in the future accidentally "reorders" them:)